### PR TITLE
FIX: account for expanded API in stats objects

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

**Comment by tacaswell** [comment]
attn @cmazzoli   I think this fixes https://github.com/NSLS-II/wishlist/issues/110

**Comment by tacaswell** [comment]
@cmazzoli What you need to do now is:

 1. read the changes
   a. if you like them hit the 'merge pull request' button on this page
   b. if you do not leave line comments asking me to change things until you are happy
 2. on your computer do a `git remote update` (this may require re-exporting https_proxy)
 3. `git merge --ff-only origin/master`
 4. restart ipython

**Comment by klauer** [comment]
Adding the setup call should (hopefully) fix most of the ca timeouts.

**Comment by cmazzoli** [comment]
Tom, I had a look but I don't understand. So I assume that they are ok and I merge + update locally in order to test.
Thank you.

**Comment by tacaswell** [comment]
@cmazzoli Did this fix your problems?

File: profile_collection/startup/20-detectors.py [hunk 1]
```diff
@@ -272,6 +272,7 @@ def __init__(self, name, pv_basename, num_points = 1000000):
         self._pv_wfrm_ts = epics.PV("{}TSTimestamp".format(pv_basename), auto_monitor=False)
         self._cb = None
         self.done = True
+        self.success = True
```
Comment (tacaswell) [inline profile_collection/startup/20-detectors.py]: @cmazzoli Add this line to the `WaveformrCollecter` class above. as well.

**Pull request merged**